### DISCUSSION TITLE
Github Copilot should specify different prices by country

### DISCUSSION THREAD

**Discussion by inie0722** [community/community#19117] [Copilot Conversations]
I want to use this project very much, but the fee of 10$ is not a small expense in developing countries. I suggest that can learn from YouTube and have different prices in different countries.

**Comment by rauzerfabiano** [comment]
Yeah, 1$ in brazil is 5,16BRL

**Reply by imxintian** [reply]
10$ is too expensive

**Reply by Fusseldieb** [reply]
A fair price would be between $3-$5 maximum.

I already hear people saying: "But what is a $5 difference gonna change". 

Well, $5 in developed countries is almost nothing, really. I know that. $5 more or less almost doesn't make a difference, maybe as a tip or something but that's it. 

The problem that in our country, for example, 1 USD is 5 BRL. With 5 BRL you do almost as much as 5 USD (buying a cola or taking a bus, for example). 

With this in mind and to take it into perspective: This is almost as paying $50. Would you pay this much on Copilot? I certainly wouldn't.

**Reply by NandoMB** [reply]
The prices should be for each country… The reality of them is different, so, it’s fair to have different prices too…

**Comment by inie0722** [comment]
In China, 10$ is about 67 RMB, which is enough for a family of three to eat for a day.
Membership of video sites in China is generally 20 RMB per month, which is about 3$.
China's per capita income is considered to be a good level in developing countries, and other developing countries may have 2-3 days of food expenses.

**Reply by whenk7** [reply]
虽然你说的没毛病，但是感觉你搁这写作文呢

**Reply by inie0722** [reply]
> 虽然你说的没毛病，但是感觉你搁这写作文呢

我是一本正经的提供建议

**Reply by imxintian** [reply]
现在已经不能用了吧？

**Reply by inie0722** [reply]
> 现在已经不能用了吧？

GitHub Copilot is free until August 22 官网写着 免费到8月22

**Reply by twelife** [reply]
> 
这么写的 但是我授权不上了，一直等待中

**Reply by badcodebuilder** [reply]
Support that, 20CNY/month is affordable
资瓷，一个月20CNY还是可以接收的

不过这个还是得巨硬自己建模算利润然后定价

**Reply by Danyalkasiri** [reply]
****

**Comment by whenk7** [comment]
是的，太贵了。



---Original---
From: ***@***.***&gt;
Date: Wed, Jun 22, 2022 11:06 AM
To: ***@***.***&gt;;
Cc: "Wenk ***@***.******@***.***&gt;;
Subject: Re: [github-community/community] Github Copilot should specifydifferent prices by country (Discussion #19117)




  
虽然你说的没毛病，但是感觉你搁这写作文呢
  
我是一本正经的提供建议

**Comment by ljt1018168** [comment]
how about `Tabnine` ?
Is there a basic (**free** ) version for everyone ?

**Comment by psstpakGithub** [comment]
At least it should offer means of payment other than credit card/banks, which simply are not available on poor countries
It should accept payment in cryptocurrency

**Reply by inie0722** [reply]
I don't know if US companies support the use of cryptocurrencies, at least in some countries cryptocurrencies are illegal, and the exchange rate between cryptocurrencies and the US dollar is too volatile.

**Reply by psstpakGithub** [reply]
> at least in some countries cryptocurrencies are illegal,

Everything is illegal at least in some countries 

> and the exchange rate between cryptocurrencies and the US dollar is too volatile.

Non an issue. Cryptocurrencies can be used only as a means of payment, not necessarily to be stored by the vendor.
Also, many cryptocurrencies like USDT or BUSD have close to zero volatility.

**Comment by Fusseldieb** [comment]
I live in Brazil and consider _$10_ too much. If it was _$3-$5_ (R$ 19,99 or R$ 29,99) I would subscribe to it immediately. But _R$60_ is too much.

**Comment by DimitrisAthanasiadis** [comment]
Additionally, those that have been using it and-at the same time-feeding GitHub with data, should receive a discount. If not a standard discount at least according to the time that they have been using it. I find it unfair that all the users pay the same, both new and existing Copilot users.

**Reply by Fusseldieb** [reply]
Afaik those who contribute to big open-source repos gain access to Copilot for free. Or at least that's what I've heard.

**Reply by jacob-8** [reply]
Yes, that's true.

**Comment by Danyalkasiri** [comment]
****

**Comment by meowtech** [comment]
Agreed! Honestly Copilot is a bit overpriced in less-developed country. Take China for example, 10$ equals to 68rmb which almost covers five meals. Copilot is great but its price makes me think twice.

**Comment by iXingo** [comment]
Can not agree more!!!

**Comment by THEOFariaP** [comment]
Agreed.
